docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+---
+version: "2"
+services:
+  goofys:
+    image: "camptocamp/goofys"
+    cap_add:
+      - SYS_ADMIN
+      - MKNOD
+    privileged: true
+    devices:
+      - "/dev/fuse"
+    environment:
+      BUCKETNAME: "c2c-s3fs"
+      AWS_ACCESS_KEY_ID: "AKIAIV5EZYBOLXQ6AFCA"
```

Comment:
haven't looked at this in details yet, but I hope these aren't your real s3 credentials